core/templates/I18nFooter.ts

```diff
@@ -103,3 +103,14 @@ angular.module('oppia').config([
       .forceAsyncReload(true);
   }
 ]);
+
+import { Directive, ElementRef, Injector } from '@angular/core';
+import { UpgradeComponent } from '@angular/upgrade/static';
+@Directive({
```

Comment:
Use a newline above*

Why is this a directive and not a component?

Reply:
Because `UpgradeComponent` requires a directive. https://angular.io/guide/upgrade#using-angularjs-component-directives-from-angular-code

Reply:
Thanks for the explanation, @srijanreddy98 !

@pankajdahiyapd: Add a newline above*